pycc/local.py

```diff
@@ -797,3 +808,120 @@ def filter_res(self, r1, r2):
             t2[i,j] = self.Q[ij] @ X @ self.Q[ij].T
 
         return t1, t2
+
+    def _trans_integrals(self, o, v):
+        """
+        Transforming all the necessary integrals to the semi-canonical PNO basis, stored in a list with length occ*occ
+        Naming scheme will have the tensor name with _ij such as Fov_ij
+  
+        Notes
+        -----
+        There are some integral transformation that may not be needed, double check and remove 
+        """
+
+        trans_intstart = time.time()
+
+        #Initializing the transformation matrices
+        Q = self.Q
+        L = self.L
+
+        #contraction notation i,j,a,b typically MO; A,B,C,D virtual semicanonical PNO;
+
+        #Initializing transformation and integral lists
+        QL = []
+        QLT = []
+
+        Fov_ij = []
+        Fvv_ij = []
+
+        ERIoovo_ij = []
```

Comment:
What permutational symmetries are lost following the transformations?  Does ovoo =/= ooov?  Does oovo =/= ooov?  (Genuine question.)